pkg/osbuild/bootc_install_to_filesystem_stage.go

```diff
@@ -0,0 +1,23 @@
+package osbuild
+
+// NewBootcInstallToFilesystem creates a new stage for the
+// org.osbuild.bootc.install-to-filesystem stage.
+//
+// It requires a mount setup of "/", "/boot" and "/boot/efi" right now
+// so that the bootupd that bootc calls finds and installs all
+// required bootloader bits.
+//
+// XXXX: rename GenBootupdDevicesMounts() helper?
```

Comment:
Sure, why not.

Reply:
I looked at this again and decided to update the comment instead. We are actually doing all this so that bootupd can run (via bootc) inside the target and I couldn't think of a better way to describe what we do and validate so for now left it at `GenBootupdDevicesMounts()`. But of course I'm open for ideas :)